src/api/SFA.DAS.Provider.Events.Api.Types/PaymentStatistics.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.Provider.Events.Api.Types
+{
+    public class PaymentStatistics
+    {
+        public int TotalNumberOfPayments { get; set; }
+        public int TotalNumberOfPaymentsWithRequired { get; set; }
```

Comment:
I'm not clear from the name of this property what it is, could it be clearer?

Reply:
I've renamed it but not sure if its any clearer, its used to highlight there are difference in the figures between the payments and required payments tables (this should never be the case but when looking at the data it seems like it is!).